### PR TITLE
add floss-weekly link to webpage

### LINKED ISSUE #15: Add FLOSS-Weekly video to website
To get more informations about the Emissions-API project, we should add the FLOSS-Weekly video to the website. 

- As weblink to youTube or
- as video watchable direktly on the project website

### PR DESCRIPTION
**Pull request by TylerBird**
This patch adds the floss-weekly video link to the webpage.

As headline I choose "Publishments" if anyone suggests another name, go ahead. If we have some other publishments, it could linked here as well.

Closes #15

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -49,3 +49,10 @@ the satellite data of the Copernicus program, but who do not want to work with
 huge amounts of scientific data directly. We will provide examples and
 libraries to quickly get you started without being an expert in satellite data
 analysis.
+
+Publishments
```
Comment (lkiesow) [inline index.md]:
```suggestion
Publications
```

Comment (tibroc) [inline index.md]: I would call it 'Presentations' we would then later link other presentations we will have held at conferences.

**Review by tibroc — CHANGES_REQUESTED** [review]
I would just change the header.

File: index.md [hunk 2]
```diff
@@ -49,3 +49,10 @@ the satellite data of the Copernicus program, but who do not want to work with
...
+------------
+
+ - [Floss-Weekly Podcast Episode 555: Emissions API](https://twit.tv/shows/floss-weekly/episodes/555)
```
Comment (lkiesow) [inline index.md]:
```suggestion
 - [FLOSS Weekly podcast episode 555: Emissions API](https://twit.tv/shows/floss-weekly/episodes/555)
```

**Review by lkiesow — CHANGES_REQUESTED** [review]

**Pull request merged**